bundles/pom.xml

```diff
@@ -60,7 +60,8 @@
     <module>org.openhab.binding.avmfritz</module>
     <module>org.openhab.binding.benqprojector</module>
     <module>org.openhab.binding.bigassfan</module>
-    <module>org.openhab.binding.bluetooth</module>
+    <module>org.openhab.binding.bloomsky</module>
+	<module>org.openhab.binding.bluetooth</module>
```

Comment:
It seems you used on the bluetooth line a "\t" instead of the usual four spaces, thus the spotless check failing.

Reply:
Thanks for the help  @dalgwen  - so simple...